tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -4089,11 +4089,6 @@ lldbg_emit_common_block_mdnode(LL_DebugInfo *db, SPTR sptr)
   cmnblk_mdnode = lldbg_create_common_block_mdnode(
       db, scope_modnode, ll_get_md_null(), SYMNAME(sptr));
```

Comment:
Do we still need a null metadata node for the optional `declaration` field?

Reply:
Sorry. I am not sure whether I got this comment correctly. Do you mean we should get rid of this field as it is unused ?

Reply:
My understanding was that this null node is meant for the `declaration` field in a `DICommonBlock` metadata, which according to `LLParser.cpp` is not a required field. So I was just asking if `lldbg_create_common_block_mdnode` could be modified to not require this parameter, since the actual argument is always `ll_get_md_null()`.

Reply:
Thanks @bryanpkc for clarification.
Yes, correct. I have updated patch for this.